src/components/MotivationForm.tsx

```diff
@@ -68,7 +64,7 @@ const MotivationForm = ({
 
     POST({ ...formData }).then((res) => {
       console.log(res);
-      if (res.message === "Yayay") {
```

Comment:
Hey @pettiboy, why is this change required?

Reply:
`api/post.ts` is directly returning string instead of `request`.